pilot/pkg/networking/plugin/authz/util.go

```diff
@@ -175,3 +100,12 @@ func extractActualServiceAccount(istioServiceAccount string) string {
 	}
 	return actualSA
 }
+
+func found(key string, list []string) bool {
```

Comment:
nit: change `found` to `isStringInList`?

Reply:
will update soon in follow-up PRs, thanks